libs/nx-remotecache-s3/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@pellegrims/nx-remotecache-s3",
   "main": "src/index.js",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
there's no need to update the version manually, it will automatically be increased after we merge on master and changes are detected in this package